src/kixi/heimdall/handler.clj

```diff
@@ -18,8 +18,8 @@
   (edn/read-string (slurp f)))
 
 (def auth-config
-  (get-config (or (util/file-exists? (System/getProperty "user.home") ".heimdall.auth-conf.edn")
-                  (io/resource (:auth-conf env)))))
+  (util/get-config (or (util/file-exists? (System/getProperty "user.home") ".heimdall.auth-conf.edn")
```

Comment:
should get-config live in kixi.heimdall.config meaning that util could be dropped? (I'm generally against util workspaces for things that aren't truly domainless)